CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # Change Log
 
+## [0.23.0](https://github.com/eclipse/lemminx/milestone/36?closed=1) (December 15, 2022)
+
+### Enhancements
+
+ * Codelens, completion, definition, documentLink, highlighting, references, rename support in RNG file. See [#1400](https://github.com/eclipse/lemminx/pull/1400).
+ * Surround selection with XML element. See [#1389](https://github.com/eclipse/lemminx/pull/1389).
+ * Switch experimental formatter to default. See [#1377](https://github.com/eclipse/lemminx/issues/1377).
+ * Full support for `xml.format.maxLineWidth` with experimental formatter. See [#1248](https://github.com/eclipse/lemminx/issues/1248).
+ * Adjust selection range for wrap element when empty selection is in start or end tag.. See [#1392](https://github.com/eclipse/lemminx/pull/1392).
+ * Support `files.trimTrailingWhitespace` setting with experimental formatter. See [#1310](https://github.com/eclipse/lemminx/issues/1310).
+ * Remove `xml.format.preserveEmptyContent` for experimental formatter. See [#1346](https://github.com/eclipse/lemminx/issues/1346).
+ * Support `xml.format.maxLineWidth` with invalid attribute values. See [#1363](https://github.com/eclipse/lemminx/pull/1363).
+ * Extend `xml.format.maxLineWidth` setting to support attributes and tag elements. See [#1359](https://github.com/eclipse/lemminx/pull/1359).
+ * Improve text content formatting in experimental formatter. See [#1331](https://github.com/eclipse/lemminx/issues/1331).
+
+### Performance
+
+ * Improve formatting performance with `xml.format.grammarAwareFormatting` setting. See [#1368](https://github.com/eclipse/lemminx/issues/1368).
+
+### Bug Fixes
+
+ * NPE in CodeLens with empty XML file. See [#1396](https://github.com/eclipse/lemminx/issues/1396).
+ * Surround with tags fails when there is no root element. See [#1395](https://github.com/eclipse/lemminx/issues/1395).
+ * Generate grammar fails on first attempt with experimental formatter. See [#1382](https://github.com/eclipse/lemminx/issues/1382).
+ * XML Completion element apply generate invalid XML content. See [#1373](https://github.com/eclipse/lemminx/issues/1373).
+ * Prevent exception in `DomElementFormatter.formatEndTagElement()`. See [#1361](https://github.com/eclipse/lemminx/issues/1361).
+ * Fix unexpected behavior of `xml.format.preservedNewlines` with experimental formatter. See [#1341](https://github.com/eclipse/lemminx/pull/1341).
+ * Autoclose tag generates an unexpected >. See [#1221](https://github.com/eclipse/lemminx/issues/1221).
+
+### Build
+
+ * Add Windows PR verification job with github actions. See [#1311](https://github.com/eclipse/lemminx/issues/1311).
```

Comment:
... and technically you also migrated the verification job entirely to GH actions. It doesn't happen on the Jenkins instance anymore. You could also add that fact in if you like.